packages/backend/src/peripherals/database/WithdrawableAssetRepository.ts

```diff
@@ -63,6 +66,19 @@ export class WithdrawableAssetRepository extends BaseRepository {
     return results[0]!.id
   }
 
+  async getWithdrawableAssetsByStarkKey(
+    starkKey: StarkKey
+  ): Promise<{ assetHash: AssetHash; balanceDelta: bigint }[]> {
```

Comment:
Why just returning this? Why not whole WithdrawableAssetRecord and then map it in controller?